README.md

```diff
@@ -1,14 +1,16 @@
 # Sqlalchemy Hawq
 
+[![Build Status](https://travis-ci.org/bcgsc/sqlalchemy_hawq.svg?branch=master)](https://travis-ci.org/bcgsc/sqlalchemy_hawq)
+
 - [Getting Started](#getting-started)
-    - [Install (For developers)](#install-for-developers)
-    - [Run Tests](#run-tests)
-    - [Deploy (For developers)](#deploy-for-developers)
-- [Using in a SQLAlchemy Project](#using-in-a-sqlalchemy-project)
-    - [How to incorporate sqlalchemy-hawq](#how-to-incorporate-sqlalchemy-hawq)
-    - [Hawq-specific table arguments](#hawq-specific-table-arguments)
-    - [Example of hawq table arguments with declarative syntax](#example-of-hawq-table-arguments-with-declarative-syntax)
-- [Using Partitions](#using-partitions)
+  - [Install (For developers)](#install-for-developers)
+  - [Run Tests](#run-tests)
+  - [Deploy (For developers)](#deploy-for-developers)
+- [Using in a SQLAlchemy project](#using-in-a-sqlalchemy-project)
+  - [How to incorporate sqlalchemy-hawq](#how-to-incorporate-sqlalchemy-hawq)
+  - [Hawq-specific table arguments](#hawq-specific-table-arguments)
+  - [Example of hawq table arguments with declarative syntax](#example-of-hawq-table-arguments-with-declarative-syntax)
+- [Using partitions](#using-partitions)
```

Comment:
The alignment of [ seems to be off here

Reply:
looks fine from what I can see, which part looks off?

Reply:
Not sure whether its my browser, but the opening [ on lines 9 and 13 seem under indented

Reply:
Oh I see, they are just different levels of titles. They are supposed to look like that. Take a look at the README as it renders on the web page

https://github.com/bcgsc/sqlalchemy_hawq/tree/feature/set-up-ci